packages/client/mutations/BatchArchiveTasksMutation.ts

```diff
@@ -0,0 +1,80 @@
+import graphql from 'babel-plugin-relay/macro'
+import {commitMutation} from 'react-relay'
+import {StandardMutation} from '../types/relayMutations'
+import ITask from '../../server/database/types/Task'
+import {BatchArchiveTasksMutation as TBatchArchiveTasksMutation} from '../__generated__/BatchArchiveTasksMutation.graphql'
+import getTagsFromEntityMap from '../utils/draftjs/getTagsFromEntityMap'
+import handleUpsertTasks from './handlers/handleUpsertTasks'
+import handleRemoveTasks from './handlers/handleRemoveTasks'
+
+graphql`
+  fragment BatchArchiveTasksMutation_tasks on BatchArchiveTasksSuccess {
+    tasks {
+      id
+      content
+      tags
+    }
+  }
+`
+
+const mutation = graphql`
+  mutation BatchArchiveTasksMutation($taskIds: [ID!]!) {
+    batchArchiveTasks(taskIds: $taskIds) {
+      ... on ErrorPayload {
+        error {
+          message
+        }
+      }
+      ...BatchArchiveTasksMutation_tasks @relay(mask: false)
+    }
+  }
+`
+
+const BatchArchiveTasksMutation: StandardMutation<TBatchArchiveTasksMutation> = (
+  atmosphere,
+  {taskIds},
+  {onError, onCompleted}
+) => {
+  return commitMutation<TBatchArchiveTasksMutation>(atmosphere, {
+    mutation,
+    variables: {
+      taskIds
```

Comment:
+1 I haven't seen this pattern elsewhere in the app, but it works either way. I'd say this would be more consistent:

```
  variables,
  {onError, onCompleted}
) => {
  return commitMutation<TBatchArchiveTasksMutation>(atmosphere, {
    mutation,
    variables,
    updater: (store) => {
      const {taskIds} = variables
```